terra-workspace-manager-jmeter/src/main/java/automation/jmeter/CreateWorkspaceSimulation.java

```diff
@@ -0,0 +1,307 @@
+package automation.jmeter;
```

Comment:
I understand that there's refactor work coming in the future. However, I think it's worth separating the test itself from all the config stuff as part of this very PR.
Currently, I'm not able to fully evaluate this POC, as it's not clear to me how much work is involved when writing new tests.

Reply:
Let's have a separate discussion on this. I agreed, there needs to be refactoring to make writing JMeter test more code efficient.